pubsub/democlient/main.go

```diff
@@ -27,21 +28,24 @@ func main() {
 		return
 	}
 
-	// Create the websocket connection.
-	origin := "/"
-	ws, err := websocket.Dial(cfg.URL, "", origin)
+	backend := slog.NewBackend(os.Stdout).Logger("PSCL")
+	backend.SetLevel(slog.LevelDebug)
+	psclient.UseLogger(backend)
+
+	// Create the pubsub client, opening a connection to the URL.
+	ctx, cancel := context.WithCancel(context.Background())
+	opts := psclient.Opts{
+		ReadTimeout:  3 * time.Second,
+		WriteTimeout: 3 * time.Second,
```

Comment:
Websocket options must be provided at time of construction because the Client immediately starts up receiving messages with these options.